docker-compose.yaml

```
@@ -1,7 +1,8 @@
 version: '3.9'
 services:
-  app:
-    container_name: app
+  gptchatbot:
+    image: bitovi/slack-bot-gptchat:latest
```

```suggestion
image: bitovi/slack-bot-gptchat:0.1.0
```

Let's do a release after the main merge (if the main merge is successful)